### DISCUSSION TITLE
Unavailable values on H3-8.0-E

### DISCUSSION THREAD

**Discussion by lis0r** [StealthChesnut/HA-FoxESS-Modbus#107] [Q&A]
Hi,

I set this up with the H3 configuration, targeting a USR-W610, and it mostly seems to be working. A handful of values have a red exclamation mark and "Unavailable" against them, though:

- Battery current
- Battery voltage
- Inv Current R
- Inv Current S
- Inv Current T
- Inverter Model

Have I set something up wrong such that it's preventing me from reading these values?

I see the occasional log lines like:
> [pymodbus.transaction] No response received to unbounded read !!!!
> [homeassistant.components.modbus.modbus] Pymodbus: HA-FoxESS-Modbus-LAN: Modbus Error: [Input/Output] No Response received from the remote unit/Unable to decode response

But largely the transactions seem to complete successfully.

Do I perhaps need to change some further settings on the W610?

Thanks!

**Comment by lis0r** [comment]
I think this might be because, in all cases except Inverter Model, "slave: 247" is missing from the yaml.

**Reply by Malcs91TF** [reply]
I also found several Unavailable values using modbusLAN.yaml .  I've added "slave: 247" in the missing locations and now those points are reading correctly.  So all I have missing from the overview are load power and Load Sum but I think that is because CT2 is not installed.
